File: mvc/src/main/java/com/bootcamp/mvc/model/Ticket.java
package com.bootcamp.mvc.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Ticket {
    
    private Long id;
    private Long id_wallet;
    private String name_token;
    private double amount;
    private boolean positive;
}

File: SegurityAngular-master/src/app/model/Ticket.ts
export class Ticket{
    id?:number;
    amount:number;
    name_token:string;
    id_wallet:number;
    positive:number;

     constructor(amount: number,name_token:string,id_wallet:number,positive:number){
        this.amount=amount;
        this.id_wallet=id_wallet;
        this.name_token=name_token;
        this.positive=positive;
       }
}